ts/features/itwallet/common/utils/itwCredentialUtils.ts

```diff
@@ -46,3 +47,41 @@ export const generateTrustmarkUrl = (
   });
   return `${verifierUrl}?${queryParams}`;
 };
+
+export const borderColorByStatus: { [key in ItwCredentialStatus]: string } = {
+  valid: IOColors.white,
+  invalid: IOColors["error-600"],
+  expired: IOColors["error-600"],
+  expiring: IOColors["warning-700"],
+  jwtExpired: IOColors["error-600"],
+  jwtExpiring: IOColors["warning-700"]
+};
+
+export const tagPropsByStatus: { [key in ItwCredentialStatus]?: Tag } = {
+  invalid: {
+    variant: "error",
+    text: I18n.t("features.itWallet.card.status.invalid")
+  },
+  expired: {
+    variant: "error",
+    text: I18n.t("features.itWallet.card.status.expired")
+  },
+  jwtExpired: {
+    variant: "error",
+    text: I18n.t("features.itWallet.card.status.verificationExpired")
+  },
+  expiring: {
+    variant: "warning",
+    text: I18n.t("features.itWallet.card.status.expiring")
+  },
+  jwtExpiring: {
+    variant: "warning",
+    text: I18n.t("features.itWallet.card.status.verificationExpiring")
+  }
+};
```

Comment:
We should move this to `itwStyleUtils`, I think it's more appropriate. What do you think?